### PR TITLE
Read-only editor for bitcoin script

### LINKED ISSUE #766: Chapter 6 Code introduction Page
Implement the Page that introduces a transaction block

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=6398%3A17173&mode=dev

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #766 and creates a new component for a read only editor without languages tab

### CODE REVIEW

**Review by daviroo — CHANGES_REQUESTED** [review]
It doesn't appear to work on mobile view and also on desktop I can see some visual bugs (screenshot below & highlighted in red)

image

**Comment by 0tuedon** [comment]
> It doesn't appear to work on mobile view and also on desktop I can see some visual bugs (screenshot below & highlighted in red)
> 
> image

Alright checking this out

**Review by benalleng — CHANGES_REQUESTED** [review]
I don't think we want to render it as a `<Lesson>` since this breaks it on mobile, lets just try to render it as a flex-wrap row so that on mobile it drops below? Or if we do want tabs we need to get rid of execute on this page

**Comment by 0tuedon** [comment]
> I don't think we want to render it as a `<Lesson>` since this breaks it on mobile, lets just try to render it as a flex-wrap row so that on mobile it drops below? Or if we do want tabs we need to get rid of execute on this page

Yh I did figure that out, i'd prefer removing the execute on the page?

File: ui/lesson/ScriptingChallenge/PlainEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          {
+            'hidden md:flex': !isActive,
+            'flex ': isActive,
+          }
+        )}
+      >
+        <MonacoEditor
+          loading={<Loader className="h-10 w-10 text-white" />}
+          width={isSmallScreen ? '100vw' : 'calc(100vw / 2)'}
+          height={`calc(var(--dynamic-height) - ${totalHeight}px)`}
```
Comment (benalleng) [inline ui/lesson/ScriptingChallenge/PlainEditor.tsx:59]: I think we should try to change the height to look a little better, still need to brainstorm how to get this just right as I tried just using `100%` but I'm not totally happy with it

File: ui/lesson/ScriptingChallenge/PlainEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        'editor.background': '#00000000',
+        'editor.lineHighlightBorder': '#00000000',
+      },
+    })
+  }
+  return (
+    <div className="read-only-editor grow border-white/25 md:max-w-[50vw] md:basis-1/3 md:border-l">
+      <div
+        className={clsx(
+          'relative grow bg-[#00000026] font-mono text-sm text-white',
```
Comment (benalleng) [inline ui/lesson/ScriptingChallenge/PlainEditor.tsx:52]:
```suggestion
          'relative grow bg-[#00000026] pt-4 font-mono text-sm text-white',
```

**Review by benalleng — CHANGES_REQUESTED** [review]
Last tiny change to add padding above the code

**Review by benalleng — APPROVED** [review]

**Review by daviroo — APPROVED** [review]

**Pull request merged**